package.json

```diff
@@ -47,6 +47,8 @@
     "babel-plugin-istanbul": "^4.1.1",
     "babel-plugin-version-inline": "^1.0.0",
     "babel-preset-minify": "^0.4.0-alpha.caaefb4c",
+    "buble": "^0.19.3",
+    "buble-loader": "^0.5.0",
```

Comment:
Remove this and use babel-loader like everybody else.

Reply:
@Pessimistress updated